pages/common/zdiff.md

```diff
@@ -0,0 +1,12 @@
+# zdiff
+
+> Invoke `diff` on gzipped files.
+> More information: <https://manned.org/zdiff>.
+
+- Compare two files, uncompressing them if necessary:
+
+`zdiff {{path/to/file1.txt.gz}} {{path/to/file2.txt.gz}}`
```

Comment:
```suggestion
`zdiff {{path/to/file1.gz}} {{path/to/file2.gz}}`
```
I think `.txt` extension isn't required over here.
[Reference](https://www.geeksforgeeks.org/zdiff-command-in-linux-with-examples/)